template/.eslintrc.js

```diff
@@ -44,6 +44,22 @@ module.exports = {
       'js': 'never',
       'vue': 'never'
     }],
+    // disallow reassignment of function parameters
+    // disallow parameter object manipulation except for specific exclusions
+    'no-param-reassign': ['error', {
```

Comment:
This would have to be wrapped in a handlebars conditional `{{#if_eq lintConfig 'airbnb'}}`

Reply:
It already is. Check line 41.

Reply:
Oh. Sometimes I'm stupid :)

Reply:
I'll merge this and then remove some of the node-spcific exceptions, because this is a frontend-template, and you are likely not gonna write express routes in it - that would be something for a SSR-focussed template.